packages/commons/tsconfig.eslint.json

```
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["src/**/*.ts", "src/**/*.js", ".eslintrc.js"]
+}
```

```suggestion
}

```
No new Line Lint errror